src/util/RouterUtil.js

```diff
@@ -127,6 +127,8 @@ function (Okta, OAuth2Util, Util, Enums, BrowserFeatures, Errors, ErrorCodes) {
   fn.routeAfterAuthStatusChange = function (router, res) {
     // Other errors are handled by the function making the authClient request
     if (!res || !res.status) {
+      // clear lastAuthResponse when response is empty
```

Comment:
nit: no need for the comment, it seems with the method name should be self explanatory...